core/tests/protractor_desktop/playTranslations.js

```diff
@@ -0,0 +1,113 @@
+    explorationEditorTranslationTab =
+      explorationEditorPage.getTranslationTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    libraryPage = new LibraryPage.LibraryPage();
+
+    users.createUser('testTranslations@translations.com', 'testTranslations');
+    users.login('testTranslations@translations.com');
+    workflow.createExploration();
+
+    explorationEditorMainTab.setStateName('First');
+    explorationEditorMainTab.setContent(forms.toRichText(
+      'This is the first card.'
+    ));
+    explorationEditorMainTab.setInteraction('EndExploration');
+    explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
+    explorationEditorTranslationTab.changeLanguage('Hindi');
+    explorationEditorTranslationTab.openUploadAudioModal();
+    explorationEditorTranslationTab.uploadAudio(
+      '../data/cafe.mp3');
+    explorationEditorPage.saveChanges();
+    explorationEditorPage.navigateToSettingsTab();
+    explorationEditorSettingsTab.setTitle('Test Translations');
+    explorationEditorSettingsTab.setCategory('Languages');
+    explorationEditorSettingsTab.setLanguage('English');
+    explorationEditorSettingsTab.setObjective(
+      'Testing if translations works');
+    explorationEditorPage.saveChanges('Done.');
+    workflow.publishExploration();
+    users.logout();
+  });
+
+  it('should play and pause audio translations', function() {
+    users.login('testTranslations@translations.com');
+    libraryPage.get();
+    libraryPage.playExploration('Test Translations');
+    explorationPlayerPage.clickAudioBar();
+    explorationPlayerPage.playAudio();
+    explorationPlayerPage.expectAudioToBePlaying();
+    explorationPlayerPage.pauseAudio();
+    explorationPlayerPage.expectAudioToBePaused();
+    users.logout();
+  });
+
+  it('should play translations for multiple languages', function() {
+    users.login('testTranslations@translations.com');
+    creatorDashboardPage.get();
+    creatorDashboardPage.editExploration('Test Translations');
+    explorationEditorPage.navigateToTranslationTab();
+    explorationEditorTranslationTab.exitTutorial();
+    explorationEditorTranslationTab.changeLanguage('Arabic');
+    explorationEditorTranslationTab.openUploadAudioModal();
+    explorationEditorTranslationTab.uploadAudio(
+      '../data/cafe.mp3');
+    explorationEditorPage.saveChanges('Added another translation');
+    libraryPage.get();
+    libraryPage.playExploration('Test Translations');
+    browser.refresh();
+    explorationPlayerPage.clickAudioBar();
+    explorationPlayerPage.playAudio();
+    explorationPlayerPage.expectAudioToBePlaying();
+    explorationPlayerPage.pauseAudio();
+    explorationPlayerPage.changeLanguage('Arabic');
+    explorationPlayerPage.playAudio();
+    explorationPlayerPage.expectAudioToBePlaying();
```

Comment:
Would it be possible to upload different mp3s for different languages and then check if correct mp3 is playing?

Reply:
@krishnarao22 What about this?